packages/lib-user/src/components/shared/BarChart/BarChart.js

```diff
@@ -102,7 +102,7 @@ function BarChart({
               const time = number / dateRangeLabel.time  
               return (
                 <Text data-testid='timeLabel'>
-                  {`${time.toFixed(0)} ${dateRangeLabel.timeLabel}`}
+                  {`${time.toFixed(1).toLocaleString()} ${dateRangeLabel.timeLabel}`}
```

Comment:
This can't use the `convertStatsSecondsToHours` utility function as sometimes it's minutes sometimes it's hours, depending on the date range. Nonetheless this change keeps the time displayed consistent with other components (with one decimal place).